includes/wikia/ExternalUser_Wikia.php

```diff
@@ -216,7 +216,7 @@ public function authenticate( $password ) {
 		Wikia\Logger\WikiaLogger::instance()->debug(
 			'NEW_HASHING ExternalUser authenticate called',
 			[
-				'user_id' => $userId,
+				'user_id' => $this->getId(),
 				'caller' => wfGetCaller(),
 				'exception' => new Exception()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exception" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)